specification/batch/data-plane/Microsoft.Batch/2017-09-01.6.0/BatchService.json

```diff
@@ -12605,17 +12719,88 @@
+          "description": "The virtual network must be in the same region and subscription as the Azure Batch account. The specified subnet should have enough free IP addresses to accommodate the number of nodes in the pool. If the subnet doesn't have enough free IP addresses, the pool will partially allocate compute nodes, and a resize error will occur. The 'MicrosoftAzureBatch' service principal must have the 'Classic Virtual Machine Contributor' Role-Based Access Control (RBAC) role for the specified VNet. The specified subnet must allow communication from the Azure Batch service to be able to schedule tasks on the compute nodes. This can be verified by checking if the specified VNet has any associated Network Security Groups (NSG). If communication to the compute nodes in the specified subnet is denied by an NSG, then the Batch service will set the state of the compute nodes to unusable. For pools created via virtualMachineConfiguration only 'Microsoft.Network/virtualNetworks' are supported. If the specified VNet has any associated Network Security Groups (NSG), then a few reserved system ports must be enabled for inbound communication. For pools created with a virtual machine configuration, enable ports 29876 and 29877, as well as port 22 for Linux and port 3389 for Windows. For pools created with a cloud service configuration, enable ports 10100, 20100, and 30100. Also enable outbound connections to Azure Storage on port 443. For more details see: https://docs.microsoft.com/en-us/azure/batch/batch-api-basics#virtual-network-vnet-and-firewall-configuration",
```

Comment:
What does "only 'Microsoft.Network/virtualNetworks' are supported" mean?  (Especially given that we already state the required form of the subnet ID.)  Are you saying that for PaaS we can use arbitrary other providers?  Can we really guarantee this?

Reply:
It should read: "Only ARM virtual networks (Microsoft.Network/virtualNetworks) are supported" I have fixed it.

Reply:
What puzzled me was the reference to virtualMachineConfiguration, as if the restrictions were different between PaaS and IaaS.  Is that removed in your fixed version?

Reply:
Oh - the restrictions _are_ different between PaaS and IaaS. So it's not removed because it's the truth.

Reply:
So do we need to say anything about what is permitted for PaaS?  Because we state in the title field that the subnet must be of the form `.../{provider}/virtualNetworks/...` - so does PaaS allow a different provider?  Any arbitrary provider as long as it has `virtualNetworks` under it?  Surely not...

I'm sorry if I'm being dense or missing something obvious.  I'm not being deliberately obtuse, genuinely confused what is being said here.

Reply:
Lets sync up offline and then you can help me determine if we need to change this text to clarify it's meaning.